lib/ansible/modules/cloud/ovirt/ovirt_templates.py

```diff
@@ -64,6 +64,29 @@
         description:
             - "Boolean indication whether to allow partial registration of a template when C(state) is registered."
         version_added: "2.4"
+    vnic_profile_mappings:
+        description:
+            - "Mapper which maps an external virtual NIC profile to one that exists in the engine when C(state) is registered.
+               vnic_profile is described by the following dictionary:
+              - C(source_network_name): The network name of the source network.
+              - C(source_profile_name): The prfile name related to the source network.
+              - C(target_profile_id): The id of the target profile id to be mapped to in the engine."
+        version_added: "2.5"
+    cluster_mappings:
```

Comment:
Please add specific dictinary keys description also for `cluster_mappings`, `role_mappings` and `domain_mappings.`

Reply:
done